### PR TITLE
remove telemetry prop: common.source

### PR DESCRIPTION
**Pull request by joaomoreno**

### CODE REVIEW

File: src/vs/platform/telemetry/test/browser/telemetryService.test.ts [hunk 1]
```diff
@@ -96,156 +95,148 @@ suite('TelemetryService', () => {
...
-		}
-
-		override publicLogError2<E extends ClassifiedEvent<OmitMetadata<T>> = never, T extends IGDPRProperty = never>(eventName: string, data?: StrictPropertyCheck<T, E>): Promise<any> {
-			return this.publicLogError(eventName, data as ITelemetryData);
-		}
+		// join(): Promise<any> {
+		// 	return Promise.all(this.promises);
+		// }
+
+		// override publicLog(eventName: string, data?: ITelemetryData): Promise<void> {
```
Comment (lramos15) [inline src/vs/platform/telemetry/test/browser/telemetryService.test.ts:218]: What's with all this commented out tests?

File: src/vs/workbench/services/telemetry/browser/workbenchCommonProperties.ts [hunk 2]
```diff
@@ -29,7 +29,7 @@ export async function resolveWorkbenchCommonProperties(
 	productIdentifier?: string,
 	removeMachineId?: boolean,
 	resolveAdditionalProperties?: () => { [key: string]: any }
-): Promise<{ [name: string]: string | boolean | undefined }> {
+): { [name: string]: string | boolean | undefined } {
```
Comment (lramos15) [inline src/vs/workbench/services/telemetry/browser/workbenchCommonProperties.ts:32]: `ICommonProperties`?

**Review by lramos15 — CHANGES_REQUESTED** [review]
Just some small nits.

Also what about init that we use in `extHostTelemetry` for common properties? Is that effected?

File: src/vs/platform/telemetry/test/browser/telemetryService.test.ts (hunk 1 above)
Comment (joaomoreno) [reply src/vs/platform/telemetry/test/browser/telemetryService.test.ts:218]: Forgot to actually delete all this unnecessary code.

File: src/vs/workbench/services/telemetry/browser/workbenchCommonProperties.ts (hunk 2 above)
Comment (joaomoreno) [reply src/vs/workbench/services/telemetry/browser/workbenchCommonProperties.ts:32]: Yes! Good eye!

**Comment by joaomoreno** [comment]
> Also what about init that we use in `extHostTelemetry` for common properties? Is that effected?

Unsure what you mean here?

**Comment by lramos15** [comment]
> > Also what about init that we use in `extHostTelemetry` for common properties? Is that effected?
> 
> I'm not sure I understand what you mean.

With your moving around of `telemetryInfo` I just want to make sure that https://github.com/microsoft/vscode/blob/dc0285fa1c3c87f0a11e876f4fc4c3a52e2cca34/src/vs/workbench/api/common/extHostTelemetry.ts#L103-L104 is unaffected. It looks like it is.

**Review by lramos15 — APPROVED** [review]

**Pull request merged**